content-scripts/cs.js

```diff
@@ -77,7 +82,11 @@ function injectUserstylesAndThemes({ userstyleUrls, themes, isUpdate }) {
       let css = theme.styles[styleUrl];
       // Replace %addon-self-dir% for relative URLs
       css = css.replace(/\%addon-self-dir\%/g, chrome.runtime.getURL(`addons/${theme.addonId}`));
-      if (isUpdate && css.startsWith("/* sa-autoupdate-theme-ignore */")) continue;
+      if (
+        isUpdate &&
+        (css.startsWith("/* sa-autoupdate-theme-ignore */") || css.startsWith("/* sa-theme-enable-ignore */"))
```

Comment:
What does `/* sa-theme-enable-ignore */"` do?

Reply:
Shouldn't enable userscript when addon enabled if that comment it there

Reply:
I don't think I understood what it does, can we have a proper written explanation?